<launch>

	<!-- Joy node -->
	<node respawn="true" pkg="joy" type="joy" name="joy" >
		<param name="dev" type="string" value="/dev/input/js0" />
	</node>

	<!-- Axes and scaling -->
	<param name="axis_linear" value="1" type="int"/>
	<param name="axis_angular" value="3" type="int"/>
	<param name="scale_linear" value="0.20" type="double"/>
	<param name="scale_angular" value="0.8" type="double"/>
	<param name="button_turbo" value="7" type="int"/>
	<param name="button_stop" value="6" type="int"/>
	<param name="button_feature1" value="1" type="int"/>
	<param name="button_feature2" value="2" type="int"/>
	
	<!-- At last - teleoperation node -->
	<node pkg="simple_apps" type="elektron_piotrek_joy" name="telepiotrek"/>
	<node pkg="simple_apps" type="serialswitch_node" name="serialsw" output="screen" />
	<node pkg="rys_base" type="rys_base_node" name="rysbase" output="screen" />
	
</launch>
